src/reconcile/rec_track.c

```diff
@@ -473,7 +473,7 @@ __wt_ovfl_reuse_search(WT_SESSION_IMPL *session, WT_PAGE *page,
 
 	if (WT_VERBOSE_ISSET(session, WT_VERB_OVERFLOW))
 		WT_RET(__ovfl_reuse_verbose(session, page, reuse, "reclaim"));
-	return (1);
+	return (0);
```

Comment:
Would it be clearer to return `WT_NOTFOUND` if the search fails, rather than having calling code check for `addr == NULL`?